internal/storage/pebble/storage.go

```diff
@@ -80,6 +80,8 @@ func (s *Storage) get(key []byte) ([]byte, io.Closer, error) {
 func getPrefix(o interface{}) (ret string) {
 
 	switch tt := o.(type) {
+	case *[]models.Account, []models.Account:
```

Comment:
Replace this models.Account with a structure that stores balance as `*big.Int`. This will simplify a lot of things